chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -486,9 +485,9 @@ public synchronized void copyState(Scene other, boolean copyChunks) {
       height = other.height;
       backBuffer = other.backBuffer;
       frontBuffer = other.frontBuffer;
-      alphaChannel = other.alphaChannel;
       samples = other.samples;
     }
+    alphaBuffer.reset();
```

Comment:
Previously, the alpha buffer of the other scene was copied. Why was this removed?

Reply:
Scene resetting/updating is pretty complicated right now and I'm not sure if the alphaBuffer is cleared every time something changes for the camera perspective. Also when the export format changes, it might have to be recalculated now.
Better save to not keep it here and recalculate this 1 frame before export.
(But if everybody is sure that the buffer can be copied, we can bring it back -- and make the scene-updating easier to understand :)

Reply:
```suggestion
    alphaBuffer.reset(); // TODO could we copy it without resetting if the export format didn't change?
```